utils/utils_windows.go

```diff
@@ -55,6 +55,14 @@ func SignalForTermination(pid int) error {
 	return windows.GenerateConsoleCtrlEvent(syscall.CTRL_BREAK_EVENT, uint32(pid)) //nolint:gosec
 }
 
+func SignalForQuit(pid int) error {
+	// TODO(Benji): Grab a goroutine dump from "http://localhost:8080/debug/pprof/heap"
+	// but that only works when EnableWebProfile == true.
```

Comment:
Per the TODO: with my changes here, restarting a part _on Windows_ no longer outputs a goroutine dump from `viam-server`. That was a helpful feature for when `viam-server` was borked and we wanted to both restart it and see what goroutines got stuck. I'm using SIGQUIT to do this on Linux, but SIGQUIT on Windows doesn't exist. If we still want stack traces, we need another solution here, but I left it as a TODO because I wasn't sure how critical it was.